presidio-anonymizer/app.py

```diff
@@ -53,31 +52,43 @@ def anonymize():
             if not content:
                 raise BadRequest("Invalid request json")
 
-            anonymizers_config = AnonymizerRequest.get_anonymizer_configs_from_json(
-                content
+            anonymizers_config = AppEntitiesConvertor.operators_config_from_json(
+                content.get("anonymizers")
             )
-            analyzer_results = AnonymizerRequest.handle_analyzer_results_json(content)
-            anoymizer_result = self.engine.anonymize(
+            analyzer_results = AppEntitiesConvertor.analyzer_results_from_json(
+                content.get("analyzer_results"))
+            anoymizer_result = self.anonymizer.anonymize(
                 text=content.get("text"),
                 analyzer_results=analyzer_results,
-                anonymizers_config=anonymizers_config,
+                operators=anonymizers_config,
             )
             return Response(anoymizer_result.to_json(), mimetype="application/json")
 
-        @self.app.route("/decrypt", methods=["POST"])
-        def decrypt() -> Union[str, Tuple[str, int]]:
+        @self.app.route("/deanonymize", methods=["POST"])
+        def deanonymize():
             content = request.get_json()
             if not content:
                 raise BadRequest("Invalid request json")
-            decrypted_text = self.decryptor.decrypt(
-                key=content.get("key"), text=content.get("text")
+            text = content.get("text")
+            deanonymize_entities = AppEntitiesConvertor.deanonymize_entities_from_json(
+                content)
+            deanonymize_config = AppEntitiesConvertor.operators_config_from_json(
+                content.get("deanonymizers"))
+            deanonymized_response = self.deanonymize.deanonymize(
+                text=text, entities=deanonymize_entities, operators=deanonymize_config
             )
-            return jsonify(result=decrypted_text)
+            return Response(deanonymized_response.to_json(),
```

Comment:
@SharonHart ?

Reply:
Generally jsonify is the preferred way, and with a shorter syntax since you don't have to state the return type. 
When the returned payload is complex, Response is shorter since you can use your own desirialization (to_json).
We can use Response everywhere, but I wanted to use is as less as possible since it's less modern

Reply:
so @balteravishay WDYT? I don't mind going either way and changing it.